### PR TITLE
Merging v8 docs to master

### PR DESCRIPTION
**Pull request by sowmyaraj92**

### CODE REVIEW

File: redirects.json [hunk 1]
```diff
@@ -1790,5 +1790,9 @@
 	{
 		"from": "/docs/integrations/available-integrations/hipchat/",
 		"to": "/docs/integrations/intro-integrations/"
+	},
+	{
+		"from": "/docs/administration/migrating-to-v7/",
+		"to": "/docs/administration/upgrading-to-v8/"
```
Comment (tristandenyer) [inline redirects.json:1796]: There are 5 or 6 other places in this redirects file that point to `"to": "/docs/administration/migrating-to-v7/"` that should also be updated to `"to": "/docs/administration/upgrading-to-v8/"` so we don't create a redirect chain.

File: src/pages/docs/administration/upgrading-to-v8.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+* Use role-based permissions which come with a fine-grained access management of some of Postman’s prominent features.
+
+* Access new features and updates because new feature development will be based on [Postman v8](https://www.postman.com/downloads/release-notes/).
+
+> Once your team migrates to Postman v7 or above, the native apps 5.x and 6.x versions and Chrome app 5.x versions will stop syncing and all [collaboration features](https://learning.postman.com/docs/collaborating-in-postman/collaboration-intro) will stop working and all members of the team need to update their apps to Postman v7 or above to continue collaborating.
+
+**Note:** The migration decision is restricted to the team admin. If you are a Postman team member(non-admin) and are not on Postman v8, please contact your admin to migrate to Postman v8.
+
+<img alt="non-admin user" src="https://assets.postman.com/postman-docs/non-admin-migrate-v8.jpg"/>
```
Comment (tristandenyer) [inline src/pages/docs/administration/upgrading-to-v8.md:70]: Not a PR-stopper, but in the future, all alt tags should be descriptive of what the image is. Example for this image could be "Dialog box for v6 to v8 users telling them to contact their admin. Screen shot."

**Review by tristandenyer — CHANGES_REQUESTED** [review]
We need to clean up the rest the v7 redirects as well.

File: redirects.json (hunk 1 above)
Comment (tristandenyer) [reply redirects.json:1796]: Resolved in #2881

**Review by claireriley — APPROVED** [review]
Looks good to me! I'll add a note about image description to the style guide.

**Review by tristandenyer — APPROVED** [review]
LGTM

**Pull request merged**